app.js

```diff
@@ -0,0 +1,59 @@
+// class Book {
+//   constructor (title, author) {
+//     this.title = title;
+//     this.author = author;
+//   }
+// }
+// const awesomeBooks = [];
+
+// awesomeBooks.push(new Book(title.value, author.value));
+
+// console.log(awesomeBooks);
```

Comment:
- [ ] Kindly remove this commented code because they are making your code not look clean :thinking: